src/server/verification/verificationAPI.js

```diff
@@ -40,7 +40,7 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
 
       try {
         const processor = createEnrollmentProcessor(storage)
-        await processor.enqueueDisposal(enrollmentIdentifier, signature, log)
+        await processor.enqueueDisposal(user, enrollmentIdentifier, signature, log)
```

Comment:
why do we need the user record?

Reply:
https://github.com/GoodDollar/GoodServer/blob/56-delete_user_facial_record_after_24_hours/src/server/db/mongo/models/delayed-task.js#L22

delayed tasks are linked to the users
for enrollment 24h removal this foreign key isn't used but I've added it to the tasks queue model for the future usage in other cases (as I've projected the queue architecture  more universal)

Reply:
user should be optional.
in the case of faceverification we dont want to keep a link between a user and his facerecord due to privacy